gorm/utilities.go

```diff
@@ -214,3 +213,9 @@ type EmptyFieldPathError struct {
 func (e *EmptyFieldPathError) Error() string {
 	return fmt.Sprintf("Empty field path is not allowed")
 }
+
+var namingStrategy = schema.NamingStrategy{}
```

Comment:
Is this required? It looks unused (supposed to be used in L220?), but I know sometimes these statements do more than they seem.